### PR TITLE
TCEC Season 21 Superfinal submission

### PR DESCRIPTION
**Pull request by kiudee**
Changes & Rationale
---------------------
- Network bumped to 69626 (from 69146). Most recent networks are very close in strength and differences in Elo can mostly be attributed to noise. A robust choice is to pick networks with the highest moving average strength (69626 or 69637), which counteracts the noise somewhat. 
- ~~Applied CPuct scaling settings of the 30s+0.3s tune.~~ Reverted due to the discovered fleet performance problems.
- Increased MaxPrefetch to MinibatchSize (896), since it clearly outperformed in a tune.
- CacheHistoryLength set to 0. This was the result of the MaxPrefetch tune.
- ~~MLH slope was reduced to 0.0001. This is also the result of a 30s+0.3s tune on V100s. A test on 10s+0.1s showed that the default slope (0.004) works slightly better (~8 Elo difference), but it is unclear how sensitive this is to time control.~~ Reverted due to the discovered fleet performance problems.
- ~~SmartPruningFactor increased to 2.5 in conjunction with using SmartPruningQEffect of 0.7. These values are not fully tuned and are still uncertain. A tune for the smooth time manager + SPF + SPFQE is currently ongoing. To synthesize the current results into robust values the marginal optimum values were chosen, which should work well with a variety of TM settings.~~ Reverted due to the discovered fleet performance problems.
- Remove the drawscore settings. There have been no robust tunes of the drawscore settings against a stronger opponent. Even then, tuning for raw strength could yield different settings than tuning for overall win%. I see no reason to experiment here.
- Ram limit has been increased to 96000, since LCZero is the only GPU engine in the Superfinal.
- Enabled SmartPruningQEffect with a small value of 0.2, which is very safe and likely beneficial.

Todo
-----
(Elo estimates are uncertain and based on shorter time controls)
- [x] Set the final network to be submitted. (2-5 Elo)
- [x] Compile branch to be used for the final. Contenders for pull requests to be used:
   + [x] [PR1488](https://github.com/LeelaChessZero/lc0/pull/1488): Don’t count transposition nodes for the time manager
   + [x] [PR1534](https://github.com/LeelaChessZero/lc0/pull/1534): Adjust smart pruning factor according to Q difference with previous move.
   + [x] [PR1596](https://github.com/LeelaChessZero/lc0/pull/1596): Quickfix for smooth TM nps misprediction.
   + [x] ~[PR1601](https://github.com/LeelaChessZero/lc0/pull/1601): Hyperpruning aka “Leela unhinged”.~
   + [ ] More?
- [ ] Adjust important settings for the engine:
   + [x] CPuct-scaling settings (~0-10 Elo)
   + [x] MLH settings (~10 Elo)
   + [x] Time manager + settings (? Elo)
   + [x] Adapt memory limit for Sufi
   + [x] Increase MaxPrefetch and use CacheHistoryLength of 0
   + [x] ~~Increase memory limit further? (Ask Aloril how much memory is needed for TB cache)~~
   + [x] Go back to default cpuct values?
   + [x] Are the MLH parameters really better?
   + [ ] More?

Todo for testing
--------------------
- [ ] Is the MaxPrefetch setting stable in benchmarks?

### CODE REVIEW

**Comment by Kovax007** [comment]
69626 or 69677!  69637 has some struggles, despite being top selfelo net

**Comment by Kovax007** [comment]
What is the value of "MovesLeftScaledFactor" ?

**Comment by SkylerLaughs** [comment]
Where we stand as of commit 5819683 updating the network to 69626 makes good sense to me, and I don’t have any real desire to change much from where the submission stands with these latest changes.

Turning the draw score settings off completely makes good sense given our collective disagreements on how best to proceed there.

As far as MLH goes, I also think it’s very safe to presume that with a low enough TC more aggressive MLH slope will show an advantage due to fewer blunders on increment in an endgame, so the 10s + .1s result really shouldn’t dissuade us any in reducing it. It’s to be expected. Clearly if the tune at 30s + .3s already shows a reduced slope is better than default contrary to the 10s + .1 TC, then we have no reason to think this isn’t even more true at something like TCEC SuFi TCs. Heck, I would probably still just turn it off completely, but a NPS benchmark there with it on and off on the backend would still be instructive.

I am sure you already know my opinion on network selection. That being that 69626 makes the most sense given self-play Elo trends.

With regard to TM, I don’t think there is much of a win to be had there at this point changing anything. Outside of updating the binary with the fix for smooth that is. Don’t think there was really a single instance in DivP where Leela TM was to blame for a loss or failing to convert a win.

Long story short, if we want to see what speed up we get from disabling MLH than great, otherwise gtg as is now.

**Comment by kiudee** [comment]
We discovered that there were surprising fluctuations in performance on the fleet workers. This can negatively impact the tunes done on the fleet. To be safe, we therefore choose to revert most parameter settings produced on the fleet until we know more.
